src/Components/Notes/NotesNavBar.jsx

```diff
@@ -58,15 +61,15 @@ export default function NotesNavBar() {
           },
         }}
       >
-        {selectedNoteId &&
+        {selectedNoteId && !isCreateNoteActive &&
          <TooltipIconButton
            title='Back to the list'
            placement='bottom'
            onClick={() => {
              removeHashParams(window.location, NOTE_PREFIX)
              setSelectedNoteId(null)
            }}
-           icon={<BackIcon/>}
+           icon={<BackIcon style={{width: '16px', height: '16px'}}/>}
```

Comment:
Please either adjust this as the theme or remove custom sx.  These end up affecting the group height and then alignment of the whole component and it's very hard to debug if they're defined all over the codebase.